src/query/useLatestLedger.ts

```diff
@@ -1,12 +1,19 @@
+import { NetworkHeaders } from "@/types/types";
 import { SorobanRpc } from "@stellar/stellar-sdk";
```

Comment:
should we use `rpc.Server` https://stellar.github.io/js-stellar-sdk/module-rpc.Server.html#.Options

Reply:
Yes, I'll update. :+1: